esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -46,6 +46,8 @@
 #define FILE_DESPRAY_URL    "FileDesprayAccess"
 #define WUDETAILS_REFRESH_MINS 1
 
+static unsigned DROPZONEFILESEARCH_MAXFILES = 1000;
```

Comment:
I would use a macro \#define for this, or if it is a variable I would change case to be dropZoneFileSearchMaxFiles.

Reply:
I may change it to: unsigned DropZoneFileSearchMaxFiles = 1000;